docs/data-sources/project_issue.md

```diff
@@ -0,0 +1,94 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "gitlab_project_issue Data Source - terraform-provider-gitlab"
+subcategory: ""
+description: |-
+  This data source allows you to access GitLab Project Issues.
+  Upstream API: GitLab API docs https://docs.gitlab.com/ee/api/issues.html
+---
+
+# gitlab_project_issue (Data Source)
+
+This data source allows you to access GitLab Project Issues.
+
+**Upstream API:** [GitLab API docs](https://docs.gitlab.com/ee/api/issues.html)
+
+## Example Usage
+
+```terraform
+data "gitlab_project" "foo" {
+  id = "foo/bar/baz"
+}
+
+data "gitlab_project_issue" "welcome_issue" {
+  project = data.gitlab_project.foo.id
+  iid     = 1
+}
+
+output "welcome_issue_web_url" {
+  value = data.gitlab_project_issue.web_url
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **iid** (Number) The internal ID of the project's issue (requires administrator or project owner rights)
```

Comment:
Is this true for reading or just creating?

Reply:
only for setting it, clarified.